docs/index.rst

```diff
@@ -50,7 +50,7 @@ minimum-tokens
 .. note::
 
   To help test case parsing, make sure that each test case docstring has the
-  tokens prefixed with ``@`` and suffixed with ``:``.  Otherwise, you may see
+  tokens prefixed with ``:`` and suffixed with ``:``.  Otherwise, you may see
```

Comment:
To help test case parsing, make sure that each test case docstring has the tokens in the following format:

token prefix
    This is configurable and by default, it is ``:``. 
token suffix
    This is not configurable and should always be ``:``.